Added a consensus RFC (#997)
tari-project/tari: RFC/src/Glossary.md

 Tari transactions is set purely by the block height of the block they are recorded in. 

## Block Header
[block header]: #block-header "A collection of information validating the information within a [block]"

> A collection of information validating the information within a [block].

Reviewer: ```suggestion
A data structure that validates the information contained in a [block].
```
